apps/summon-app/src/utils/content.ts

```diff
@@ -2,51 +2,51 @@ export const FORM_COPY = {
   },
 };
 export const INFO_COPY = {
   VOTING_STK:
-    "Voting stake tokens are created when the DAO is summoned. What does your community want to name the DAO's token?",
+    "The voting stake token name is set when the DAO is summoned. What does your community want to name the DAO's token?",
   TOKEN_SYMBOL:
-    'This is the common shorthand referring to stake among DAO members.  What abbreviated symbol would your DAO like to use here?',
+    'This is the common shorthand your DAO will use to refer to voting stake among DAO members. What abbreviated symbol would your DAO like to use here?',
   STAKE_TRANSFER:
-    'Allow or disallow the voting stake tokens to be transferable between addresses.',
+    'Should DAO members be allowed to transfer their voting stake to other accounts? After summoning, the DAO can change this with a proposal.',
   NV_STAKE_TRANSFER:
-    'Allow or disallow the non-voting stake tokens to be transferable between addresses',
+    'Should DAO members be allowed to transfer their non-voting stake to other accounts. After summoning, the DAO can change this with a proposal.',
   VOTING_PERIOD:
-    'The length of time a proposal will remain available for voting stake members to submit their vote',
+    'The length of time a proposal will remain open for voting stake members to submit votes.',
   GRACE_PERIOD:
-    'The time period between a proposal’s approval and its execution.  This provides time for members to ragequit their shares, before the approved proposal is executed.',
+    'The time period between a proposal’s approval and its execution. This provides time for members to ragequit their shares, before the approved proposal is executed.',
   QUORUM:
-    'The minimum percentage of DAO members needed to approve a proposal for it to execute.  This percentage can greatly affect DAO operations',
+    'The minimum percentage of DAO members (voting stake) needed to approve a proposal for it to be executed. This percentage can greatly affect DAO operations, though the DAO can change it with a proposal after summoning with a proposal.',
   MIN_RETENTION:
-    'Used as a safety measure, this is the minimum percentage of DAO members needed to ragequit for a proposal to automatically fail',
+    'If less than this percentage of DAO members (voting and non-voting stake) remain in the DAO after the Grace period for a given proposal, that proposal will not be executed.',
   SPONSOR_THRESHOLD:
-    'The minimum amount of voting stake that a member needs to have their proposal automatically sponsored upon creation.',
+    'The minimum number of voting stake tokens that a member needs to sponsor a proposal.',
   NEW_OFFERING:
-    'Intended to protect against spam issues, this is the fee required to submit a DAO proposal.',
+    'Intended to protect against spam issues, this is the fee required to submit a DAO proposal. Any fees go into the DAO treasury.',
```

Comment:
Any fees? Just starting the sentence with 'fees' helps the reader understand which fees we're talking about, imo.